tests/cases/compiler/classFieldSuperNotAccessible.ts

```diff
@@ -0,0 +1,12 @@
+// @target: esnext
+class T {
+    field = () => {}
+}
+class T2 extends T {
+    f() {
+        super.field()
+        //    ~~~~~ error here
```

Comment:
Copying the error span into the test is actually a little confusing when reading the error baseline. Maybe just a single caret?